bitnami/rabbitmq/templates/secrets.yaml

```diff
@@ -1,29 +1,18 @@
 {{- if or (not .Values.auth.existingErlangSecret) (not .Values.auth.existingPasswordSecret) }}
+{{- $secretName := include "common.secrets.name" (dict "existingSecret" .Values.auth.existingSecret "context" $)}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "rabbitmq.fullname" . }}
+  name: {{ $secretName }}
   namespace: {{ .Release.Namespace | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
   {{- if .Values.commonAnnotations }}
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 type: Opaque
 data:
-  {{- if not .Values.auth.existingPasswordSecret }}
-  {{- if .Values.auth.password }}
-  rabbitmq-password: {{ .Values.auth.password | b64enc | quote }}
-  {{- else }}
-  rabbitmq-password: {{ randAlphaNum 10 | b64enc | quote }}
-  {{- end }}
-  {{- end }}
-  {{- if not .Values.auth.existingErlangSecret }}
-  {{- if .Values.auth.erlangCookie }}
-  rabbitmq-erlang-cookie: {{ .Values.auth.erlangCookie | b64enc | quote }}
-  {{- else }}
-  rabbitmq-erlang-cookie: {{ randAlphaNum 32 | b64enc | quote }}
-  {{- end }}
-  {{- end }}
+  rabbitmq-password: {{ include "common.secrets.passwords.manage" (dict "secret" $secretName "key" "rabbitmq-password" "length" 16 "providedValues" (list "auth.password") "context" $) }}
```

Comment:
Again, you will need to follow the same approach used in Odoo:

https://github.com/bitnami/charts/blob/28661447b627cc8d3d25e7e1ee79fec21d687cff/bitnami/odoo/templates/secrets.yaml#L16-L18

Reply:
Also, note that the old length was `10` and not `16`.

Reply:
> Also, note that the old length was `10` and not `16`.

That last bit was intentional. A 10-char password struck me as odd.